src/emailconfiguration.cc

```diff
@@ -87,9 +87,9 @@ replace_tokens (
 static std::string
 s_generateEmailBodyResolved (fty_proto_t *alert, const std::string& extname)
 {
-    char *result_char = translation_get_translated_text (BODY_RESOLVED.c_str ());
-    std::string result (result_char);
-    zstr_free (&result_char);
+    std::string result = getIpAddr();
+    result += translation_get_translated_text (BODY_RESOLVED.c_str ());
```

Comment:
Memory leak, value returned by `translation_get_translated_text()` must be freed with `free()` or, preferably, use ZstrGuard.